### ISSUE TITLE
Match date parameter with date in database

### ISSUE THREAD

**Issue by karan9nov** [tkaczmarzyk/specification-arg-resolver#116]
Hey! 

I am trying to match the passed date `(yyyy-MM-dd)` in the parameter with the ISO format date saved in the database. But unfortunately, I am not getting any responses. Just wanted to see if there is something I am missing. 

My Spec looks like
```
@And({
        @Spec(path = "orderDate", params = "orderDate", config = "yyyy-MM-dd", spec = Equal.class)
})
public interface OrderSpec extends Specification<Order> {}
```

Entity class `Order` contains a column `orderDate`
```
@Column(name = "order_date")
@Temporal(TemporalType.TIMESTAMP)
private Date orderDate;
```

The current code results in an empty list of results. Is there something I am missing? 

Could you please help out? Thanks!

**Comment by hasanli-orkhan** [comment]
Have you resolved this problem? I have the same issue. I need to match LocalDate (yyyy-MM-dd) to LocalDateTime

**Comment by tkaczmarzyk** [comment]
Partial support added in https://github.com/tkaczmarzyk/specification-arg-resolver/pull/161  (introduced EqualDay)

**Comment by tkaczmarzyk** [comment]
Extended support (matching date with ignoring time) delivered in https://github.com/tkaczmarzyk/specification-arg-resolver/pull/193

**Comment by tkaczmarzyk** [comment]
Released in v2.16.0
